docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -439,6 +439,8 @@ The following configuration creates a CI job that runs an SCA scan using Semgrep
 </TabItem>
 </Tabs>
 
+> Note: It is necessary to create a context ('dev' in the snippet) to declare the SEMGREP_APP_TOKEN environment variable. "Organization Settings" -> "Contexts" -> "Create Context" -> "Add Environment Variable". 
```

Comment:
@sebasrevuelta Thanks for outlining the additional steps! I think we shouldput it in the steps (beginning at line 390) and reminding the user of it in the snippet). I'll make a separate branch so you can see my changes easily. Thanks again! 🙏